.github/workflows/publish-docs.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
+        go:
+          - 1.17
```

Comment:
Is this using go?

Reply:
I used the `update-go-version.sh` and looks like https://github.com/code-ready/crc/blob/main/update-go-version.sh#L18-L20 is culprit to adding it.

Reply:
updated